cmd/prometheus/main.go

```diff
@@ -101,11 +101,15 @@ func main() {
 
 		prometheusURL string
 
-		logLevel promlog.AllowedLevel
+		promlogConfig promlog.Config
 	}{
 		notifier: notifier.Options{
 			Registerer: prometheus.DefaultRegisterer,
 		},
+		promlogConfig: promlog.Config{
+			Level:  new(promlog.AllowedLevel),
```

Comment:
We shouldn't have to list these here, as that goes against the goal of not having to update code when there's a new setting.

Reply:
Oh right; I just made a PR for promlog's `AddFlags` method that also initializes these for the config, which avoids this issue.

See prometheus/common#159.